script/brave_license_helper.py

```diff
@@ -84,6 +84,11 @@ def AddBraveCredits(prune_paths, special_cases, prune_dirs, additional_paths):
             "URL": "https://github.com/brave/hashset-cpp",
             "License": "MPL-2.0",
         },
+        os.path.join('brave', 'vendor', 'web-discovery-project'): {
```

Comment:
@fmarier is this needed since it's our code?

Reply:
Until it's fully bundled into brave-core, I think it makes sense to include it here.